governance-services/latest/install/uninstall.md

```diff
@@ -56,7 +56,7 @@ $ java -jar bin\alfresco-mmt.jar uninstall <ModuleId> <WARFileLocation>
     ```bash
     Module 'alfresco-rm-enterprise-share' installed in 'tomcat\webapps\share.war'
     -    Title:        Alfresco Record Management Share Extension
-    -    Version:      7.1
+    -    Version:      7.2
     -    Install Date: Tue Sept 28 08:50:15 UTC 2021
```

Comment:
_Future update_: we should consider making lines 42 & 60 more generic or update the date